cache/redisCache.js

```diff
@@ -61,14 +61,22 @@ if (featureToggles.isFeatureEnabled('enableRedisConnectionLogging')) {
 const subPerspectives = [];
 const pubPerspectives = [];
 rconf.instanceUrl.pubsubPerspectives.forEach((rp) => {
-  const s = redis.createClient(rp, opts);
-  s.subscribe(rconf.perspectiveChannelName);
-  subPerspectives.push(s);
+  // Only create subscribers here if we're doing real-time events from main app
+  if (!process.env.REALTIME_APPLICATION) {
+    const s = redis.createClient(rp, opts);
+    s.subscribe(rconf.perspectiveChannelName);
+    subPerspectives.push(s);
+  }
+
   pubPerspectives.push(redis.createClient(rp, opts));
 });
 
-const subBot = redis.createClient(rconf.instanceUrl.pubsubBots, opts);
-subBot.subscribe(rconf.botChannelName);
+// Only create subscribers here if we're doing real-time events from main app
+let subBot;
+if (!process.env.REALTIME_APPLICATION) {
```

Comment:
Any reason for not using a toggle?

Reply:
Updated to set up and use toggle @jgraff2